backend/utils/emailer.js

```diff
@@ -45,82 +46,102 @@ function optionsForItem(item) {
   return options
 }
 
-async function getEmailTransporter(shop) {
-  let networkConfig = {}
+/**
+ * Returns config and email transporter object
+ * @param {Model.Shop} shop DB model
+ * @returns {{
+ *  transporter,
+ *  fromEmail,
+ *  supportEmail,
+ *  shopConfig, // Decrypted shop config
+ *  configJson // Shop's public config.json file
+ * }}
+ */
+async function getEmailTransporterAndConfig(shop) {
```

Comment:
Would be great to have a unit test for this method as well since the logic to pick the config is not super straight forward. Let's perhaps not block merging this PR on writing a unit test, but would you mind adding a TODO for yourself? Sorry for being a pita on this - just trying to ensure our codebase core functionality is stable :)